raiden/api/python.py

```diff
@@ -645,6 +652,30 @@ def transfer_async(
         if not is_binary_address(target):
             raise InvalidAddress('target address is not valid.')
 
+        if secret is not None:
+            if len(secret) != 66:
```

Comment:
On what basis was this length dictated?

Reply:
the secret is 32 bytes. When you make it a hax string it would be 64 chars (since each byte represented by 2 chars .
on top of that there is the `0x` prefix which makes the string size to be 66 chars.

Reply:
You're right. Didn't think about it this way, thanks.